src/web/main.rs

```diff
@@ -38,9 +43,22 @@ fn main() {
 
         mig.maintain(awvc, 0);
         mig.maintain(karma, 0);
+
+        // assign everything to same domain
+        let domain = mig.add_domain();
+        mig.assign_domain(awvc, domain);
+        mig.assign_domain(karma, domain);
+        mig.assign_domain(vc, domain);
+        mig.assign_domain(article, domain);
+        mig.assign_domain(user, domain);
+        mig.assign_domain(vote, domain);
```

Comment:
Ok, did not know that! I thought we had to manually assign things to domains. Thanks!